tools/clang/test/CodeGenSPIRV/raytracing.target-env-error.hlsl

```diff
@@ -1,6 +1,6 @@
 // RUN: %dxc -T lib_6_3
 
-// CHECK: error: Vulkan 1.1 with SPIR-V 1.4 is required for Raytracing but not permitted to use
+// CHECK: error: vulkan1.1spirv1.4 is required for Raytracing but not permitted to use
```

Comment:
This is not a good change to the error message.

Reply:
Right, "reverted" this part. (Split the string to target env function with env to human readable).
Reverted test file changes.